examples/address_generator.py

```diff
@@ -38,13 +33,13 @@ def main(uri, index, count, security, checksum):
     # Here's where all the magic happens!
     api_response = api.get_new_addresses(index, count, security, checksum)
     for addy in api_response['addresses']:
-        print(binary_type(addy).decode('ascii'))
+        print(bytes(addy).decode('ascii'))
 
     print('')
 
 
 def get_seed():
-    # type: () -> binary_type
+    # type: () -> bytes
```

Comment:
💭 We can also convert all the type hints (possibly in this one, or create a separate PR; what do you think?).

Reply:
I would leave type annotations to a separate PR, where we get rid of type hints in favor of proper type annotations.